docs/pages/web/banneroverlay.js

```diff
@@ -0,0 +1,206 @@
+          />
+          <MainSection.Card
+            type="don't"
+            sandpackExample={
+              <SandpackExample code={dontStack} name="Don't - Stack" hideEditor hideControls />
+            }
+            description="Stack multiple UpsellOverlays; only one UpsellOverlay should appear on the screen per time."
+          />
+        </MainSection.Subsection>
+      </MainSection>
+      <MainSection name="Variants">
+        <MainSection.Subsection title="Image">
+          <MainSection.Card
+            sandpackExample={<SandpackExample code={image} name="Image" layout="column" />}
+            description="With an image for Pin or Board actions, or the Pinterest logo."
+          />
+        </MainSection.Subsection>
+
+        <MainSection.Subsection title="Avatar">
+          <MainSection.Card
+            sandpackExample={<SandpackExample code={avatar} name="avatar" layout="column" />}
+            description="With an Avatar for Profile or Pinner-related messaging."
+          />
+        </MainSection.Subsection>
+
+        <MainSection.Subsection title="Icon">
+          <MainSection.Card
+            sandpackExample={<SandpackExample code={icon} name="icon" layout="column" />}
+            description="For when an icon is needed to represent content that isn’t a pin or a profile."
+          />
+        </MainSection.Subsection>
+      </MainSection>
+      <MainSection name="Responsive">
+        <MainSection.Subsection>
+          <MainSection.Card
+            description={`UpsellOverlay is responsive to different devices. Therefore, OverlayUpsell dweb is set to a max width of 900 px to preserve a great usability experience and consistency with other components.
+
+On desktop devices, the BannerOverlay should appear at the top of the screen (below navigation). The BannerOverlay shouldn't block navigation.
+On mobile devices, the BannerOverlay should appear at the bottom of the screen. The 'offset' prop can be used to adjust the position of the BannerOverlay.`}
+          />
+          <SandpackExample code={mobile} name="Main BannerOverlay example" hideEditor />
+          <SandpackExample code={desktop} name="Main BannerOverlay example" hideEditor />
+        </MainSection.Subsection>
+      </MainSection>
+
+      <AccessibilitySection name={generatedDocGen?.displayName}>
+        <MainSection.Subsection
+          title="Labels"
+          description={`
+          \`dismissButton\` and \`primaryAction\`, require a short, descriptive label for screen readers, which should also be localized.
+
+          Icons and thumbnails on BannerOverlay are purely decorative, and can therefore have an empty string as the \`accessibilityLabel\`. The thumbnail (Image) or Icon should supply an alt or accessibilityLabel, respectively, if the Image or Icon supplies extra context or information.`}
+        />
+        <MainSection.Subsection
```

Comment:
Localization is typically it's own larger Main Section

Reply:
Done!